taxonomies/packaging_recycling.txt

```diff
@@ -30,8 +33,41 @@ nl:recyclen bij oud papier, bij oud papier, in papierbak, in de papierbak, papie
 # If the recycling instruction is to recycle in the paper bin, we can't assume the material is paper
 # it could also be cardboard
 
+# in the Netherlands for recycling of drink cartons
+<en:recycle
+nl:drankencartons
```

Comment:
For the packaging_recycling taxonomy, we should list the action as the main name:
recycle, discard, reuse etc.

So here we could have a main name as:

en:recycle-with-drink-cartons

Also the main name must be English, so that we don't need to update tests later.

Reply:
OK I changed the names.

In NL metal is thrown away with regular waste, but recovered/recycled during incineration. How shall I add this?

Reply:
This taxonomy is more about the recycling instructions than the actual recycling, so it should just be "to discard" I think.